homebrew/using_different_homebrew_formula_versions.md

```diff
@@ -0,0 +1,35 @@
+# Using different Homebrew formula versions
+
+Today, I was trying to run the executable from a gem and got the following error:
+
+```bash
+Users/jessieayoung/.rbenv/versions/2.1.3/lib/ruby/gems/2.1.0/gems/rmagick-2.13.2/lib/RMagick.rb:11:in
```

Comment:
This line is a little long, causing a scrollbar to be added to the code block. Maybe break it up a bit? I think even  just removing the "in" at the end would be enough.